src/apis/DataDisplayApi.ts

```diff
@@ -0,0 +1,24 @@
+
+import axios from 'axios';
+import { DataDisplayRequest, DataDisplayResponse } from '../types/Types';
+
+const BASE_URL = 'https://api.example.com'; // Replace with your API base URL
```

Comment:
This is the URL that we should call:
`https://0fg9aos700.execute-api.ap-south-1.amazonaws.com/asdfasdf/sample-route-key`